### ISSUE TITLE
".NET Framework/ .NET Core" dropdown and "Create workbook" button are appearing with green color border in High contrast black mode and brown color border in High contrast white mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#337]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode before launching the application.
1. Launch Xamarin Workbooks application. Which invokes "Create New Workbook" window.
2. Observe the border color of ".NET Framework/.NET Core" dropdown and "Create Workbook" button.

### Expected Behavior
Border color should follow high contrast standards
### Actual Behavior
Border color appears in green in HC Black and Brown in HC White for ".NET Framework/.NET Core" drop down and "Create Workbook" button.
### User Impact
As the border color appears in green color in HC Black and Brown in HC White, user may perceive it as disabled element which is actually not. It deviates the user from the purpose.
### Screenshots
![4 border in brown hcw](https://user-images.githubusercontent.com/22956130/40722274-8438fa84-6439-11e8-9904-f8e6349c4df2.png)
![4 border in green hcb](https://user-images.githubusercontent.com/22956130/40722275-84739860-6439-11e8-8d81-135fac31729a.png)
![4 normal mode](https://user-images.githubusercontent.com/22956130/40722276-84eeb6c6-6439-11e8-94ab-9c7538d51e89.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
